Llk/Lexer.php

```diff
@@ -300,4 +313,17 @@ protected function matchLexeme($lexeme, $regex, $offset)
             'length' => mb_strlen($matches[0])
         ];
     }
+
+    /**
+     * @param string $text
+     * @return bool
+     */
+    private function validateInputInUnicodeMode($text)
+    {
+        if (strpos($this->_pcreOptions, 'u') !== false && !mb_check_encoding($text, 'utf-8')) {
```

Comment:
Instead of dependency on mbstring, which could even produce different result than PCRE itself, I'd recommend using:
```suggestion
        if (strpos($this->_pcreOptions, 'u') !== false && preg_match('##u', $text) === 1) {
```

https://3v4l.org/VEWTT